contrib/images/bcd/docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+services:
+  babylondnode0:
+    container_name: babylondnode0
+    image: "babylonlabs-io/babylond"
+    command: >
+      babylond --home /babylondhome start --log_format 'plain' 2>&1 | tee /babylondhome/babylond.log
+    cap_add:
+      - SYS_PTRACE
+    security_opt:
+      - seccomp:unconfined
+    ports:
+      - "26656-26657:26656-26657"
+      - "1317:1317"
+      - "9090:9090"
+      - "2345:2345"
+    volumes:
+      - .testnets/node0/babylond:/babylondhome:Z
+    networks:
+      localnet:
+        ipv4_address: 192.168.10.2
+
+  babylondnode1:
+    container_name: babylondnode1
+    image: "babylonlabs-io/babylond"
+    command: >
+      babylond --home /babylondhome start --log_format 'plain' 2>&1 | tee /babylondhome/babylond.log
+    cap_add:
+      - SYS_PTRACE
+    security_opt:
+      - seccomp:unconfined
+    ports:
+      - "26666-26667:26656-26657"
+      - "1318:1317"
+      - "9091:9090"
+      - "2346:2345"
+    volumes:
+      - .testnets/node1/babylond:/babylondhome:Z
+    networks:
+      localnet:
+        ipv4_address: 192.168.10.3
+
+  ibcsim-bcd:
```

Comment:
What do you think of building a `bcd` Docker image and using a container in this
docker compose file? This way we can reuse the IBC relayer functionalities in
the e2e test suite. Probably not for this PR

Reply:
The code tagged here includes two functions that use chain configs: `runHermesIBCRelayer` and `createBabylonPhase2Channel`. If we look at their implementation, it only uses channel ID, port ID, and some IBC configurations inside chain config. If we hardcode these values is this possible?